bol/retailer/api.py

```
@@ -86,6 +89,8 @@ def ship_order_item(
                 "orderItemId": order_item_id
             }
         ]
+        if quantity is not None:
```

Ruff (0.15.21)

[error] Undefined name `quantity`

(F821)